pkg/controllers/nodeclass/garbagecollection/controller.go

```diff
@@ -137,8 +137,7 @@ func (c *Controller) Reconcile(ctx context.Context) (reconciler.Result, error) {
 	if err := c.cleanupInactiveProfiles(ctx); err != nil {
 		return reconciler.Result{}, err
 	}
-	// Requeue after 30 minutes
-	return reconciler.Result{RequeueAfter: 30 * time.Minute}, nil
+	return reconciler.Result{RequeueAfter: 5 * time.Minute}, nil
```

Comment:
QQ: Why did we drop this down from 30m?

Reply:
Alternatively we could just attempt to delete the instance profile after a failed `Create` attempt and fail open (letting the garbage collector handle the transient error cases). I don't have a strong preference either way.

Reply:
I'll go ahead and add this as a follow-up